src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/Divider.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import dividerStyles from './divider.scss';
+
+type Props = {
+    children: string,
+};
+
+export default class Divider extends React.PureComponent<Props> {
+    render() {
+        const {children} = this.props;
+        const dividerClass = classNames(
+            dividerStyles.divider,
+            {
+                [dividerStyles.noText]: !children,
```

Comment:
How about using the [`empty` pseudo selector](https://developer.mozilla.org/de/docs/Web/CSS/:empty) instead?